section1/set8/nestedLoops/challenge.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+)
+
+func main() {
+	n, err := strconv.Atoi(os.Args[1])
+
+	if err != nil {
+		fmt.Println("Provide an integer.")
+		return
+	}
+
+	fmt.Printf("%5s", "X")
+	for i := 0; i <= n; i++ {
+		fmt.Printf("%5d", i)
+	}
+	fmt.Println()
+
+	for i := 0; i <= n; i++ {
+		fmt.Printf("%5d", i)
+
+		for j := 0; j <= n; j++ {
```

Comment:
good to see that you are practicing nested loops

Reply:
this is the example program

Reply:
i am copying down what the instructor wrote

Reply:
ok